azure-migration.rst

```diff
@@ -0,0 +1,22 @@
+.. post:: August 8, 2018
+   :tags: azure, downtime, infrastructure
+   :author: David
+   :location: SAN
+
+
+Planned Move to Azure
+=====================
+
+We wanted to make you aware that on Saturday August 18 at 10:00am PDT (1:00pm EDT, 17:00 UTC),
+Read the Docs will be having a scheduled downtime of approximately 2 hours.
```

Comment:
We should give ourselves more buffer than this. If we think it should only take 2 hrs best case, we should schedule the window for at least 4, because something will go wrong, and if we end early, people are happy.